src/extension.ts

```diff
@@ -148,7 +150,10 @@ export function activate(context) {
         vscode.window.registerTreeDataProvider('extension.vsKubernetesExplorer', treeProvider),
 
         // Code lenses
-        vscode.languages.registerCodeLensProvider(HELM_REQ_MODE, new HelmRequirementsCodeLensProvider())
+        vscode.languages.registerCodeLensProvider(HELM_REQ_MODE, new HelmRequirementsCodeLensProvider()),
+
+        // Telemetry
+        new Reporter(context)
```

Comment:
`new Reporter()` but don't assign it to any variable. It feels a little wierd to me. 
Could change it to `initializeReporter(context)`?

Reply:
It is stored in the subscriptions array.  Agree it looks a bit weird because it's surrounded by expressions which are more obviously side-effectful, but I'd rather not change the name because this source is provided by the Azure VS Code SDK.  We could put a wrapper function in `extension.ts` if it feels that awkward though.

Reply:
yeah, we could wrap it to a meaningful function in extension.ts. Here if i don't read the implementation, i don't get what's for `new Reporter()`.

Reply:
Good point.  If it's not readable then we should make it readable.  Added a `registerTelemetry` function.  Thanks!